usautobuild/utils.py

```diff
@@ -104,28 +109,29 @@ def iterate_output(cmd: subprocess.Popen[bytes]) -> Iterator[tuple[str, bool]]:
 
 
 def git_version(directory: Optional[Path] = None, brief: bool = True) -> str:
-    "Get repository version for given folder in human readable format. Single line"
+    """Get repository version for given folder in human readable format. Single line"""
 
     if directory is None:
         directory = Path.cwd()
 
-    if brief:
-        fmt = "%h%d %ar"
+    repo = Repo(directory, search_parent_directories=True)
```

Comment:
why does it search on parent? the game repo folder is inside the builder folder, wouldn't it conflict?

Reply:
that's the default behavior of git executable. I'm unsure about build step because it does stuff in subfolders. this shouldn't hurt since it already worked this way

Reply:
it shouldn't conflict because it stops when it sees 1st .git folder